support/packages/local-npm/README.md

```diff
@@ -0,0 +1,213 @@
+   exactly the same configuration they will have when the packages are truly 
+   released to the public. This means, no more `yarn link`s or flaky
+   `node_modules` folders due to them.
+2. Because you are using your local npm repo, you are not polluting the public
+   repos with bogus or alpha versions.
+3. Since you are not publishing to any public repo, you may reuse the same
+   version number for a given package as many times as you want.
+4. Because of 3, there's no need to use prerelease version numbers or anything
+   like that, i.e., no more `-pre.0` and things like that needed to test
+   development packages. You can simply use the next version number you plan to
+   release.
+5. Because of 4 it is less likely that you break dependencies in the
+   `package.json` files, because you don't need to play with prerelease version
+   numbers that can be updated wrongly when the final release is made.
+6. Given that you are using true version numbers, you can keep your range semver
+   expressions (f.e.: `^1.0.0`) in your `package.json` files because they won't
+   break as they do when you use prerelease numbers (remember that prerelease
+   numbers don't match semver expressions in Node).
+7. Once you are done, or every time you want to reset the state of the repo, you
+   can delete all your prepublished local packages by simply removing them from
+   Verdaccio's `storage` folder.
+
+Note, however, that not everything is better than when using `yarn link`, being
+the main con the need to redeploy a new version each time you modify any
+package, making live development & testing impossible.
+
+For cases where such speed is needed, you may resort to `yarn link` but beware
+that you must now very well what you are doing because `yarn link` may 
+completely break node's resolution algorithm as it diverts resolution of
+dependencies of linked packages to their own folder locations, which usually
+leads to unpredictable results.
+
+You've been warned.
+
+### How to use it
+
+The `local-npm` has three commands, of which you will normally use two:
+
+1. `install`: this works like `yarn install` but it makes sure that the package
+   is re-downloaded from the repo and written again to your `node_modules`
+   folder. On the contrary, `yarn` doesn't do this unless you change the
+   dependency version number or remove `yarn.lock` and `node_modules`.
+2. `publish`: this is like `npm publish` but it makes sure you are publishing to
+   your local repo only and overwrites any published package with the same name
+   and version.
+3. `registry`: this has two subcommands for querying (`get`) the active
+   repository, or setting (`set`) it to `local` or `public`.
+
+Usually you will always have your `npm` and `yarn` repos pointing to your local
+repo. This is because Verdaccio acts as a proxy of the real public repo so, if
+you request anything it doesn't have, it will download it, cache it and serve
+it.
```

Comment:
```suggestion
Usually you will always have `npm`/`yarn` pointing to your local
registry. This is because Verdaccio acts as a proxy of the public registry so, if
you request anything it doesn't have, it will download it, cache it and serve
it.
```